doc/event-alarm-framework/event-alarm-framework.md

```diff
@@ -102,7 +94,7 @@ Such a change has an important metric called *severity* to indicate how critical
    The set of alarms and their severities are an indication to health of various applications of the system and System LED can be deduced from alarms.
    An acknowledged alarm means that operator is aware of the condition so, acknowledged alarm will be taken out of consideration.
 
-Both events and alarms get recorded in a new DB called EVENT DB in a new redis instance.
+Both events and alarms get recorded in redis DB.
```

Comment:
which redisDB? could you update if its some existing redisDB or a new one?

Also, as of now, when applications (processes/daemons) publish their event(s)/alarm(s), they do NOT get stored in any redis (or persistent DB) - right? In that case, are they stored by eventD framework/container in memory? no EVENT DB as of now?